pyproject.toml

```diff
@@ -11,10 +11,10 @@ keywords = ["artifactory"]
 
 [tool.poetry.dependencies]
 python = "^3.6"
-pydantic = "^0.23.0"
 requests = "^2.21"
 email_validator = "^1.0"
 requests_toolbelt = "^0.9.1"
+pydantic = "^1.3"
 
```

Comment:
(Adding a comment on this line because I cannot add it below)
If you upgrade the versions of mypy, black and detect-secrets in the pre-commit config, you should update them here too ^^